cmake/ubuntu-cross-aarch64.cmake

```diff
@@ -0,0 +1,13 @@
+# Target operating system / architecture.
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
+
+# C/C++ compiler.
+set(CMAKE_C_COMPILER "/usr/bin/aarch64-linux-gnu-gcc")
+set(CMAKE_CXX_COMPILER "/usr/bin/aarch64-linux-gnu-g++")
```

Comment:
Does this have to be an absolute path, could we make it more portable by just having `aarch64-linux-gnu-gcc`?